service-data-access/build.gradle

```diff
@@ -23,6 +24,13 @@ tasks.dockerPrepare.configure {
 //    mavenCentral()
 // }
 
+sourceSets {
+    integrationTest {
+        compileClasspath += sourceSets.main.output
+        runtimeClasspath += sourceSets.main.output
+    }
```

Comment:
There's a similar example: https://github.com/department-of-veterans-affairs/abd-vro/blob/c6aefb88480907c74ced6a7013381cbdb98a67e7/buildSrc/src/main/groovy/local.java.test-integration-conventions.gradle.example#L11-L13

I'm wondering if you should mimic other code in that file. 🤷

Reply:
I have no idea about buildSrc here. Looking at the sample, I prefer not to change at present. 

The whole idea for me to create this integrationTest is to avoid build break as these tests used to sit in test and keep the these testing code. Those tests in the integrationTest can be run locally and are useful to detect real problems. However, we have no idea how to run the integration test in the build process at this time. This integration test won't run in the build process.

When we know how to integrate this "integration test" in the build (CICD) process, we can revisit this to follow the sample.